boxoffice/models/line_item.py

```diff
@@ -189,6 +189,20 @@ def fetch_all_details(cls, item_collection):
         return db.session.execute(line_item_query).fetchall()
 
     @classmethod
+    def fetch_assignee_details(cls, item_collection):
```

Comment:
This should be a regular method on the `ItemCollection` class. It does not belong on the `LineItem` class. Please move it. The `fetch_all_details` classmethod should move as well.